k8s/storage/volumeattachments.go

```diff
@@ -1,31 +1,31 @@
 package storage
 
 import (
-	storagev1 "k8s.io/api/storage/v1"
+	storagebeta "k8s.io/api/storage/v1beta1"
```

Comment:
why is this version change required? Or was this the original one before refactor?

Reply:
Yes, that's the original one.